node/Cargo.toml

```diff
@@ -18,7 +18,7 @@ edition = "2021"
 
 [features]
 default = [ "parallel" ]
-parallel = [ "rayon" ]
+parallel = [ ] #TODO(kaimast): is this still used?
```

Comment:
FYI I found a while ago that although snarkVM has a very clean `serial` feature with consistent use of cfg_*, snarkOS does not at the moment